### PR TITLE
e2e - smoke tests for redis stack

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/pageObjects/add-redis-database-page.ts [hunk 1]
```diff
@@ -38,7 +39,7 @@ export class AddRedisDatabasePage {
       //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
       //-------------------------------------------------------------------------------------------
       //BUTTONS
-      this.addDatabaseButton = Selector('[data-testid=add-redis-database]');
+      this.addDatabaseButton = Selector('[data-testid=add-redis-database]', {timeout: 1000});
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/add-redis-database-page.ts:42]: We are not adding it here generally, just within the common action functions/tests

File: tests/e2e/tests/smoke/browser/edit-key-name.e2e.ts [hunk 2]
```diff
@@ -14,7 +15,11 @@ fixture `Edit Key names verification`
     .meta({ type: 'smoke' })
     .page(commonUrl)
     .beforeEach(async () => {
-        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        if(await addRedisDatabasePage.addDatabaseButton.visible) {
+            await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        } else {
+            await acceptLicenseAndConnectToRedisStack();
+        }
```
Comment (ofersteinberg) [inline tests/e2e/tests/smoke/browser/edit-key-name.e2e.ts:22]: I see it updated over many tests. Consider make a one main function that hold these statements.

File: tests/e2e/pageObjects/add-redis-database-page.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/pageObjects/add-redis-database-page.ts]: removed

File: tests/e2e/tests/smoke/browser/edit-key-name.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/smoke/browser/edit-key-name.e2e.ts]: done

File: tests/e2e/pageObjects/user-agreement-page.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ export class UserAgreementPage {
       //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
       //-------------------------------------------------------------------------------------------
       // COMPONENTS
-      this.userAgreementsPopup = Selector('[data-testid=consents-settings-popup]');
+      this.userAgreementsPopup = Selector('[data-testid=consents-settings-popup]', {timeout: 1000});
```
[the remark this answers is not in the document]
Comment (tanyatrayanava) [reply tests/e2e/pageObjects/user-agreement-page.ts]: yes, you are right, removed it too

File: tests/e2e/helpers/database.ts [hunk 4]
```diff
@@ -135,6 +128,20 @@ export async function acceptLicenseTermsAndAddOSSClusterDatabase(databaseParamet
     await myRedisDatabasePage.clickOnDBByName(databaseName);
 }
 
+//Accept License terms
+export async function acceptLicenseTerms(): Promise<void> {
+    await t.maximizeWindow();
+    await userAgreementPage.acceptLicenseTerms();
+    await t.expect(userAgreementPage.userAgreementsPopup.visible).notOk('The user agreements popup is not shown', {timeout: 5000});
+}
+//Accept License terms and connect to the RedisStack database
```
[the remark this answers is not in the document]
Comment (tanyatrayanava) [reply tests/e2e/helpers/database.ts:138]: done

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**